src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -98,46 +97,42 @@ internal sealed class SynthesizedRecordPropertySymbol : SourceOrRecordPropertySy
 
         public override SyntaxList<AttributeListSyntax> AttributeDeclarationSyntaxList => new SyntaxList<AttributeListSyntax>();
 
-        private sealed class GetAccessorSymbol : SynthesizedInstanceMethodSymbol
+        private abstract class AccessorSymbol : SynthesizedInstanceMethodSymbol
```

Comment:
>AccessorSymbol [](start = 31, length = 14)

See also [https://github.com/dotnet/roslyn/pull/44883#discussion_r443289929](https://github.com/dotnet/roslyn/pull/44883#discussion_r443289929), this might make it easier to deal with overriding.